src/api/decoding-scanned-codes.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: API Reference
+title: Decoding Scanned Codes
+permalink: /api/decoding-scanned-codes
+---
+
+# Decoding Scanned Codes
+
+A scanned code can be a either QR code or a barcode that a merchant or a patron scans.
+The scanned code can be decoded to check the provider of the code.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|    Field    |        Type        |                           Description                            |
+| :---------- | :----------------- | :--------------------------------------------------------------- |
+| code        | String             | The code that has been scanned.                                  |
```

Comment:
> The code that has been scanned

You've used the same word to describe the field. Please get more specific.

Maybe something like: The data utf8 representation of data decoded from what was scanned.

Reply:
Sure, I've taken your suggestion